### PR TITLE
Remove link tags in service form description

### PR DESCRIPTION
**Pull request by kennyt**

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @philipnrmn

File: src/js/schemas/service-schema/EnvironmentVariables.js [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
 /* eslint-disable no-unused-vars */
 import React from 'react';
 /* eslint-enable no-unused-vars */
 
 let EnvironmentVariables = {
-  description: (
-    <span>
-      Set variables for each task your service launches. You can also use variables to expose Secrets. <Link to="system-security-secrets">Manage secrets here</Link>. <a href="https://mesosphere.github.io/marathon/docs/task-environment-vars.html" target="_blank">Learn more about variables</a>.
-    </span>  
+  description: Hooks.applyFilter(
```
Comment (kennyt) [inline src/js/schemas/service-schema/EnvironmentVariables.js]: going to create this hook here just incase plugins need to change the description

File: src/js/schemas/service-schema/General.js [hunk 2]
```diff
@@ -7,7 +6,7 @@ let General = {
   title: 'General',
   description: (
     <span>
-      Configure your container service here or <Link to="universe">install from Universe</Link>.
+      Configure your container service here or <a href="#/universe">install from Universe</a>.
```
Comment (kennyt) [inline src/js/schemas/service-schema/General.js]: we cannot use Link tags because they need context

**Comment by jfurrow** [comment]
:ship:

**Comment by kennyt** [comment]
LGTM, nice one

**Pull request merged**